makejdk-any-platform.sh

```diff
@@ -41,7 +41,7 @@ fi
 if [[ "$OS_MACHINE_NAME" == "armv7l" ]] ; then
    export JVM_VARIANT=${JVM_VARIANT:-zero}
    export MAKE_ARGS_FOR_ANY_PLATFORM=${MAKE_ARGS_FOR_ANY_PLATFORM:-"DEBUG_BINARIES=true images"}
-   export CONFIGURE_ARGS_FOR_ANY_PLATFORM=${CONFIGURE_ARGS_FOR_ANY_PLATFORM:-"--with-jobs=${NUM_PROCESSORS}}"}
```

Comment:
Thanks, but I can't take credit for that.  It was Mr Jenkins who found it.
I guess nobody wondered/cared why the ARM build was failing ;-( !?